paymentsheet/src/main/java/com/stripe/android/customersheet/CustomerSheetViewModel.kt

```diff
@@ -29,6 +29,10 @@ internal class CustomerSheetViewModel @Inject constructor(
 
     private val isLiveMode = paymentConfiguration.publishableKey.contains("live")
 
+    private val _backstack = MutableStateFlow<List<CustomerSheetViewState>>(
+        value = emptyList()
+    )
```

Comment:
Would it make sense to track the entirety of screens in here instead of just the ones _below_ the current one? This would be more similar to `BaseSheetViewModel` and allow us to just derive the current screen from `_backStack`.

Reply:
Nit: Also, we can remove the `_` because there’s no public `backStack` that we expose.